modules/backend/assets/js/auth/unistall-sw.js

```diff
@@ -0,0 +1,11 @@
+// Unistall a Service Worker before a User Signs in to prevent cache issues
+navigator.serviceWorker.getRegistrations().then(
+
```

Comment:
Spacing on this is all wacky, there shouldn't be extra empty lines and there should be a space after the `for`. Comment should be `Unregister all service workers before signing in to prevent cache issues`